dashboard/app/controllers/pd/workshop_dashboard_controller.rb

```diff
@@ -4,13 +4,15 @@ class WorkshopDashboardController < ApplicationController
     def index
       @permission = nil
 
+      contact_rp = RegionalPartner.find_by(contact: current_user)
```

Comment:
I think we want any regional partner association, not just contact, i.e. any `program_manager` association from https://github.com/code-dot-org/code-dot-org/pull/13646.